retroshare-gui/src/retroshare-gui.pro

```diff
@@ -170,7 +170,7 @@ win32-g++ {
 
 	OBJECTS_DIR = temp/obj
 
-    dLib = ws2_32 gdi32 uuid ole32 iphlpapi crypt32 winmm
+    dLib = ws2_32 wsock32 gdi32 uuid ole32 iphlpapi crypt32 winmm
```

Comment:
If `wsock32` is needed only when using JSON API it should be added conditionally only when JSON API is used and it probably goes into use-libretroshare.pro not retroshare-gui

Reply:
You're right. I pushed a fix.